reconcile/aws_ami_share.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from reconcile import queries
+
+from reconcile.utils.aws_api import AWSApi
+
+QONTRACT_INTEGRATION = "aws-ami-share"
+MANAGED_TAG = {"Key": "managed_by_integration", "Value": QONTRACT_INTEGRATION}
+
+
+def filter_accounts(accounts):
```

Comment:
Type hints, please. It took me a while to understand why you're not returning `sharing_accont_names` in the first place.

Reply:
will do